crop_videos.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+'''
+Script to crop a video in files of a determined lenght.
+'''
+
+
+import subprocess
+import re
+import math
+from optparse import OptionParser
+
+
+length_regexp = 'Duration: (\d{2}):(\d{2}):(\d{2})\.\d+,'
+re_length = re.compile(length_regexp)
+
+def parse_options():
+    parser = OptionParser()
+
+    parser.add_option("-f", "--file",
+                        dest = "filename",
+                        required = true,
+                        type = str,
+                        help = "file to split, for example sample.avi",
+                        type = "string",
+                        action = "store"
+                        )
+    parser.add_option("-s", "--split-size",
+                        dest = "split_size",
+                        required = true
+                        help = "split or chunk size in seconds, for example 10",
+                        type = "int",
+                        action = "store"
+                        )
+    (options, args) = parser.parse_args()
+
+
+
+def main():
+
+    (filename, split_length) = parse_options()
+    if split_length <= 0:
+        print "Split length can't be 0"
+        raise SystemExit
+
+    output = subprocess.Popen("ffmpeg -i '"+filename+"' 2>&1 | grep 'Duration'",
+                            shell = True,
+                            stdout = subprocess.PIPE
+                            ).stdout.read()
+    print output
+    matches = re_length.search(output)
+    if matches:
+        video_length = int(matches.group(1)) * 3600 + \
+                        int(matches.group(2)) * 60 + \
+                        int(matches.group(3))
+        print "Video length in seconds: "+str(video_length)
+    else:
+        print "Can't determine video length."
+        raise SystemExit
+
+    split_count = int(math.ceil(video_length/float(split_length)))
+    print(split_count)
+    if(split_count == 1):
```

Comment:
`if split_count is 1:`  is better IMHO

Reply:
Is this because of any style guide? I can't find why this is it better than the other way  @JaledMC

Reply:
Good question. Sometimes I see how my lint gives this warning, using PEP8,  because `is` is more human readable than `==`. But I can't find the reference. Let this code without changes.